File: rabbitmq/src/test/java/test/RabbitMqTests.java
package test;

import info.xiaomo.rabbitmq.config.Sender;
import org.junit.Test;
import org.springframework.beans.factory.annotation.Autowired;

public class RabbitMqTests {

    @Autowired()
    private Sender sender;

    @Test
    public void hello() throws Exception {
        sender.send();
    }

}

File: rabbitmq/src/test/java/test/RabbitMqTests.kt
package test

import info.xiaomo.rabbitmq.config.Sender
import org.junit.Test
import org.springframework.beans.factory.annotation.Autowired

class RabbitMqTests {

    @Autowired
    private val sender: Sender? = null

    @Test
    @Throws(Exception::class)
    fun hello() {
        sender!!.send()
    }

}
